### ISSUE TITLE
is there a way to turn off warning?

### ISSUE THREAD

**Issue by ml-and-ml** [TissueImageAnalytics/tiatoolbox#848]
```
[WARNING] Metadata: Objective power inferred from microns-per-pixel (MPP).
[WARNING] Metadata: Falling back to TIFF resolution tag for microns-per-pixel (MPP).
```

Can this warning be suppressed, or flagged to not be a problem?

**Comment by shaneahmed** [comment]
You can use [Duplicate Filter](https://github.com/TissueImageAnalytics/tiatoolbox/blob/b0b14742df31ff64e84e841a0c6d73e981d99a76/tiatoolbox/__init__.py#L54). This has been implemented in #783

**Comment by ml-and-ml** [comment]
thanks!
